code/datums/controllers/process/gang.dm

```diff
@@ -110,18 +110,25 @@
 		if (!length(turfList))
 			message_admins("All attempts to find a valid location to spawn a weapons crate failed!")
 			return
-		var/obj/storage/crate/gang_crate/guns_and_gear/crate = new(pick(turfList))
-		broadcast_to_all_gangs("<span style='font-size:24px'> We've dropped off weapons & ammunition at <b>\the [drop_zone.name]!</b> It's anchored in place for 5 minutes, so get fortifying!</span>")
+		var/turf/location = pick(turfList)
+		var/datum/client_image_group/imgroup = get_image_group(CLIENT_IMAGE_GROUP_GANG_OBJECTIVES)
+		var/image/objective_image = image('icons/effects/gang_overlays.dmi', location, "cratedrop")
```

Comment:
Not sure what this will do if the turf is RCDed/exploded? Might want to consider making this an `/obj/effects` with an image attached instead.